app/experimenter/experiments/api/v5/queries.py

```diff
@@ -53,7 +54,19 @@ def resolve_outcomes(root, info):
         return Outcomes.all()
 
     def resolve_targeting_config_slug(root, info):
-        return root._text_choices_to_label_value_list(NimbusExperiment.TargetingConfig)
+        return [
+            NimbusExperimentTargetingConfigSlugChoice(
+                label=member.label,
+                value=choice_name,
+                application_values=NimbusExperiment.TARGETING_CONFIGS[
+                    member.value
+                ].application_choice_names,
+            )
+            for (
+                choice_name,
+                member,
+            ) in NimbusExperiment.TargetingConfig.__members__.items()
```

Comment:
You can just loop over an Enum and get the individual entries and call `.name` `.value` on them instead of digg into private attributes